src/components/ui/Textarea.js

```diff
@@ -0,0 +1,21 @@
+import styled from 'styled-components/macro'
+
+const StyledTextarea = styled.textarea`
+  border: 1px solid ${({ theme }) => theme.secondaryBackground};
+  box-sizing: border-box;
+  border-radius: 12px;
+  padding: 15px 20px;
+  resize: none;
+  color: ${({ theme }) => theme.secondaryText};
+  font-family: ${({ theme }) => theme.fonts};
+  font-size: 18px;
+  width: 100%;
+  height: 120px;
+
+  ::placeholder {
+    color: ${({ theme }) => theme.tertiaryText};
+    font-family: ${({ theme }) => theme.fonts};
+  }
+`
```

Comment:
i would also ask siraj about validation (:valid and :invalid pseudo-classes). i think invalid boxes will just have a red border around them

Reply:
@SirajChokshi

Reply:
@SirajChokshi bump 😘

Reply:
We'll handle validation after this PR.

Reply:
It's closely related to Formik.

Reply:
nvm siraj, dw bout it